charts/seed-controlplane/charts/etcd/values.yaml

```diff
@@ -44,5 +47,19 @@ etcdResources:
     cpu: "4"
     memory: 30G
 
+sidecarResources:
+  limits:
+    cpu: 500m
+    memory: 2Gi
```

Comment:
I don't quite remember why and when we increased it to 10Gi but does it makes sense keep these values same as configured in webhook earlier? Ref: https://github.com/gardener/gardener-extensions/blob/master/pkg/webhook/controlplane/etcd.go#L88

Reply:
Done